crates/viewer/re_viewer/src/ui/welcome_screen/welcome_section.rs

```diff
@@ -10,6 +10,9 @@ pub(super) const WELCOME_SCREEN_BULLET_TEXT: &[&str] = &[
     "Configure the viewer interactively or through code",
 ];
 
+const DOC_BTN_TEXT: &str = "Go to documentation →";
+const DOC_BTN_COLOR: &str = "#60A0FF";
```

Comment:
Tried this:

```suggestion
const DOC_BTN_COLOR: Color32 = hex_color!("#60A0FF");
```

---


```
error[E0015]: cannot call non-const fn `Color32::from_rgba_unmultiplied` in constants
  --> crates/viewer/re_viewer/src/ui/welcome_screen/welcome_section.rs:14:32
   |
14 | const DOC_BTN_COLOR: Color32 = hex_color!("#60A0FF");
   |                                ^^^^^^^^^^^^^^^^^^^^^
   |
   = note: calls in constants are limited to constant functions, tuple structs and tuple variants
   = note: this error originates in the macro `hex_color` (in Nightly builds, run with -Z macro-backtrace for more info)
```